### ISSUE TITLE
Add Metamask Snap as a Cosmoskit option

### ISSUE THREAD

**Issue by mhagel** [hicommonwealth/commonwealth#5379]
## Description

Leap has created a MM snap that allows MM users to interact with Cosmos chains.

## Project Owner

@mhagel 

## Acceptance Criteria

- As a Metamask user, I can sign-in to and join Cosmos communities

## Additional context

https://docs.cosmoskit.com/integrating-wallets/leap-metamask-comos-snap
- Support unknown, but nice to have: I can vote and create onchain proposals

Blocked by #5378

**Comment by CowMuon** [comment]
How does this look with our web3modal prototype?

**Comment by CowMuon** [comment]
@RhysCommonwealth Can we get a p value on this? Which communities would be interested in this? How important is it to currently active and potentially on-boardable communities?

**Comment by CowMuon** [comment]
This was not really backlogged as Growth has/had not indicated when this is needed by, or deprioritized ?)

**Comment by RhysCommonwealth** [comment]
not sure. I haven't heard anything from our communities regarding this. I can ask around

**Comment by jnaviask** [comment]
Wont fix.
